test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,36 @@
+# Kubemark Pre-existing Provider Guide
+
+## Introduction
+
+The goal of the pre-existing provider is to land kubemark containers onto
```

Comment:
IMO we are still missing clarity wrt the distinction between "kubernetes master" and "kubemark master". You need to clearly state the following:
- There is a base kubernetes cluster already running (the normal master and the real minions). This is an assumption that is valid for all modes of running kubemark (preexisting or gce or ...). And we land kubemark containers on a pre-existing kubernetes cluster *always*.
- However, the only difference in this mode is we do not create the *kubemark* master as part of the script and assume that it has already been setup (probably some custom version setup by a developer), and simply point the script to that VM using $MASTER_IP.